pywbem_mock/_mainprovider.py

```diff
@@ -84,11 +91,9 @@ class CIMRepository(ResolverMixin, object):
     defined by the instance constructor (Currently InMemoryRepository) to store
     and retrieve data representing the CIM classes, CIM instances, and
     CIM qualifier declarations.
-
     """
 
-    def __init__(self, conn, datastore, default_namespace=DEFAULT_NAMESPACE):
-        # pylint: disable=line-too-long
+    def __init__(self, conn, datastore):
         """
         This CIM repository implementation uses a data store implementation
```

Comment:
I suggest to remove the first paragraph of the `__init__()` method and to start right with `Parameters:`. This works much better when creating documentation (which we don't for this class), since the docstring of `__init__()` is combined with the class docstring.

Reply:
DONE